pkg/kubelet/kubelet.go

```diff
@@ -690,7 +690,8 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		klet.livenessManager,
 		klet.startupManager,
 		klet.runner,
-		kubeDeps.Recorder)
+		kubeDeps.Recorder,
+		klet.clock.Now())
```

Comment:
nit: would it be a better dependency injection strategy to pass the clock interface to the manager?